invenio.cfg

```diff
@@ -421,6 +429,8 @@ if APP_ENVIRONMENT == 'local':
    'adminUV@test.com': 'adminUV'
   }
 
+  RATELIMIT_ENABLED = False
```

Comment:
This is only enabled in `local` and to allow bulk loading data via the REST API.